src/pmi/pmi.cs

```diff
@@ -1031,13 +1038,17 @@ public static int Main(string[] args)
                     }
                 }
 
-                if ((command == "PREPALL") || (command == "PREPALL-QUIET"))
+                bool all = command.IndexOf("ALL") > 0;
+                bool verbose = !(command.IndexOf("QUIET") > 0);
+                bool time = verbose || command.IndexOf("TIME") > 0;
+
```

Comment:
Seems like we should more to a more "standard" argument passing model. It looks like this parser will support something like `PREPALL-QUIETFOOBARTIME`, which is weird.

Reply:
Agree we probably should rework this, but it's not high on my list of todos.